File: Cap 06/6.2 - Matrizes de Uma Dimensao/Media Geral V2/medverv2.swift
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 6 - Topico 6.2
 * 
 * Programa ..: MEDIA GERAL - VERSAO 2
 */

import Foundation

var MD: [Float] = [Float](repeating: 0.0, count: 8)
var SOMA: Float = 0.0
var MEDIA: Float = 0.0

print("MEDIA GERAL - V2\n")

SOMA = 0.0
for I in 0...7 {
    print("Entre a nota \(I + 1): ", terminator: "")
    if let mdInput = readLine(), let md = Float(mdInput) {
        MD[I] = md
        SOMA += MD[I]
    }
}

MEDIA = SOMA / 8

print("Resultado da media = \(MEDIA)")

File: Cap 06/6.2 - Matrizes de Uma Dimensao/Media Geral V2/medverv2.go
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 6 - Topico 6.2
 * 
 * Programa ..: MEDIA GERAL - VERSAO 2
 */

package main

import (
  "fmt"
)

func main() {
	
  MD := make([]float64, 8)
  SOMA := 0.0
  
  fmt.Println("MEDIA GERAL - V2\n")
  
  for i := 0; i <= 7; i++ {
    fmt.Printf("Entre a nota %d: ", i+1)
    fmt.Scanln(&MD[i])
    SOMA = SOMA + MD[i]
  }
  
  MEDIA := SOMA / 8
  
  fmt.Printf("Resultado da media = %5.1f\n", MEDIA)
  
}


